### DISCUSSION TITLE
Error querying the database: db error: FATAL: no more connections allowed (max_client_conn)

### DISCUSSION THREAD

**Discussion by icecreamwebdev** [prisma/prisma#15349] [Q&A]
### Bug description

Hi, 

It seems my site goes down when google crawls it throwing the following error: 

ERROR	PrismaClientInitializationError: Error querying the database: db error: FATAL: no more connections allowed (max_client_conn)

I've ensured only one client is being generated using a global instance like the docs suggest: 

```
import { PrismaClient } from "@prisma/client"

let prisma

if (process.env.NODE_ENV === "production") {
  prisma = new PrismaClient()
} else {
  if (!global.prisma) {
    global.prisma = new PrismaClient()
  }

  prisma = global.prisma
}

export default prisma
```

My database postgres string is also set up t use pgbouncer:

postgres://postgres:blank@db.blank.supabase.co:1234/postgres?sslmode=require&pgbouncer=true&pool_timeout=45

Not sure why i'm getting this error, i'm guessing pgbouncer is set up incorrectly?

### Expected behavior

Prisma recycle database connections

### Prisma information

n/a

### Environment & setup

Database: supabase, postgresql
Environment: Vercel, Next js

### Prisma Version

```
prisma                  : 3.8.1
@prisma/client          : 4.2.1

```

**Comment by garrensmith** [comment]
@icecreamwebdev could you open a ticket with supabase so they can confirm if you are using their pgbouncer correctly?

**Reply by vladi-stevanovic** [reply]
👋 Hello @icecreamwebdev - I wanted to follow up to see if this is still an issue for you or if you were able to resolve it?
Let us know, we'd love to know what was the cause or if we can help!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
